app/geolocation_api_parser.py

```diff
@@ -0,0 +1,50 @@
+import json
+import pytz
+import requests
+from config import GEOCODE_API_KEY, TIMEZONE_API_KEY
+from datetime import datetime
+
+
+geocode_url = 'https://maps.googleapis.com/maps/api/geocode/json?'
+timezone_url = 'https://maps.googleapis.com/maps/api/timezone/json?'
```

Comment:
like i said, these are constants and should therefore by convention be uppercase